### DISCUSSION TITLE
Planning du cours

### DISCUSSION THREAD

**Discussion by CamilleTCH** [heig-vd-dai-course/heig-vd-dai-course#440] [Q&A] [answered]
Bonjour,

Est-ce que toutes les dates des travaux écrits et rendus des projets sont déjà connues ?

Si oui, où pourrais-je les trouver ? J'ai regardé dans les premiers chapitres et sur la page principale du cours, mais je n'ai pas trouver ce que je cherchais.

Merci d'avance !

**Comment by Syseria** [comment, marked as answer]
Hello,

Tu peux les trouver dans l'onglet Projet -> [DAI 2024-2025](https://github.com/orgs/heig-vd-dai-course/projects/2) comme vu lors de la présentation d'[intro au cours](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/01-introduction-and-course-organization/PRESENTATION.md#course-planning---calendrier-acad%C3%A9mique) chapitre 01.

L'évaluation 2 est normalement la semaine du 25.11 au 29.11 mais peut-être décaler de 1 semaine à cause du début de semestre.
Même chose pour l'évaluation 3 la semaine 13.01 au 17.01.

Bonne soirée

**Reply by ludelafo** [reply]
Merci beaucoup d'avoir répondu à la question @Syseria !

@CamilleThHEIG, vous pouvez aussi retrouver l'information dans le README du repo principal à l'adresse suivante : https://github.com/heig-vd-dai-course/heig-vd-dai-course?tab=readme-ov-file#course-planning-and-material.

Une toute bonne journée,
Ludovic
